app/main/views/marketplace.py

```diff
@@ -75,14 +75,22 @@ def terms_and_conditions():
 def get_brief_by_id(framework_framework, brief_id):
     briefs = data_api_client.get_brief(brief_id)
     brief = briefs.get('briefs')
+    brief_responses = data_api_client.find_brief_responses(brief_id=brief_id).get("briefResponses")
```

Comment:
add status kwarg to apiclient call here :arrow:
`client.find_brief_responses(brief_id = 5594, status='submitted,draft')`